Enhancing Military Candidate Selection (#3698)
DFHack/dfhack: docs/plugins/sort.rst

Squad assignment overlay
------------------------

> The squad assignment screen can be sorted by name, by migrant wave, by stress, by various military-related skills or by specific potential ratings.

Reviewer: ```suggestion
The squad assignment screen can be sorted by name, by migrant wave, by stress,
by various military-related skills or by long-term military potential.
```

If sorted by "leadership", then the citizen is sorted according to the highest
skill they have in leader, teacher, or military tactics.

If sorting is done by "melee potential" citizens are arranged based on their
melee potential rating. This rating is a heuristic measure that takes into
account genetic predispositions in physical and mental attributes, as

> well as body size. Dwarves (and other humanoid creatures) with bigger rating

Reviewer: ```suggestion
well as body size. Dwarves (and other humanoid creatures) with higher rating
```

are expected to be more effective in melee combat against opponents of
similar skill levels.

If sorting is done by "ranged potential" citizens are arranged based on their
ranged potential rating. This rating is a statistical measure that takes into
account genetic predispositions in physical and mental attributes.

> Dwarves (and other humanoid creatures) with bigger rating are expected to be

Reviewer: ```suggestion
Dwarves (and other humanoid creatures) with higher rating are expected to be
```
